qa/tasks/cephadm.py

```diff
@@ -315,12 +317,16 @@ def first_in_ceph_log(pattern, excludes):
                 run.Raw('|'), 'head', '-n', '1',
             ])
             r = ctx.ceph[cluster_name].bootstrap_remote.run(
-                stdout=StringIO(),
+                stdout=BytesIO(),
                 args=args,
+                stderr=StringIO(),
             )
-            stdout = r.stdout.getvalue()
+            stdout = r.stdout.getvalue().decode()
             if stdout != '':
                 return stdout
+            stderr = r.stderr.getvalue()
+            if stderr != '':
```

Comment:
Nit: I would get rid of explicit `!= ''` comparison for both `stdout` and `stderr` for consistency with other tasks.
```suggestion
            if stderr:
```